Parking/OffStreetParking/doc/spec.md

```diff
@@ -239,7 +239,15 @@ it conveys what is such special location.
 + `provider` : Parking site service provider.
     + Normative references: [https://schema.org/provider](https://schema.org/provider)
     + Optional
+
++ `measuresPeriod` : The measures period.
+    + Attribute type: [Number](http://schema.org/Number)
+    + Optional
     
++ `measuresPeriodUnit` : The measures period unit.
```

Comment:
please check other properties which are using units of measurement to check how is this conveyed in the data models

thanks